ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -22,7 +22,7 @@ resources:
       memory: 200Mi
 tests:
 - as: control-plane-120nodes
-  cron: 0 12 * * 2
+  cron: 0 6 * * 4
```

Comment:
For the 4.15 jobs can we get > 1 run per week?

Reply:
Sure Joe. For this PR, I only focused on ec build qualification and z stream verification.

We can visit while 4.15 run cadence in a separate PR and see which days makes sense for which workloads etc.